lib/common/watchdog.c

```diff
@@ -245,25 +245,41 @@ pcmk_locate_sbd(void)
 long
 crm_get_sbd_timeout(void)
 {
-    const char *env_value = getenv("SBD_WATCHDOG_TIMEOUT");
-    long sbd_timeout = crm_get_msec(env_value);
+    static long sbd_timeout = -1;
 
+    if (sbd_timeout == -1) {
+        sbd_timeout = crm_get_msec(getenv("SBD_WATCHDOG_TIMEOUT"));
```

Comment:
intention of the static sbd_timeout is probably that we are doing this just once as the environment isn't gonna change anyway. if env-variable isn't set we get in here over and over again.

Reply:
Good point -- -2 may be better here.